### ISSUE TITLE
Let the user define the logging level

### ISSUE THREAD

**Issue by iacolippo** [CalculatedContent/WeightWatcher#54]
Hi, thanks for creating this nice Python package :) It would be nice to be able to define the logging level desired. When running inside a larger application with other logging mechanisms, the logs coming from `weightwatcher` are pretty verbose and make it hard to parse things. I see it's already [marked as TODO](https://github.com/CalculatedContent/WeightWatcher/blob/master/weightwatcher/weightwatcher.py#L42)

Perhaps you could have a `set_logging_level` function to set a global verbose level in the library?

**Comment by charlesmartin14** [comment]
The latest version on the Trunk now supports custom logging, as

`watcher = ww.WeightWatcher(model=model, log_level=logging.INFO)`

**Comment by charlesmartin14** [comment]
This is now in minor release 0.45.  Closing this issue
